docs/manual-EN/2.query-language/4.statement-syntax/1.data-definition-statements/TTL.md

```diff
@@ -1,79 +1,111 @@
 
 ## TTL configurations
 
-The time to live value is set in seconds.
+- The `ttl_duration` is set in seconds. If it is set to -1 or 0, the vertex property in the tag does not expire.
 
-- If TTL is set, when the sum of the `ttl_col` and the `ttl_duration` is less than the current time, we consider the data as expired.
+- If TTL is set, when the sum of the `ttl_col` and the `ttl_duration` is less than the current time, we consider the the vertex property in the tag as expired.
 
-- If TTL is not set or the `ttl_col` is null, then the time to live has no effect.
-
-- If TTL is set to -1 or 0, then data in the field does not expire.
+- -When the vertex has multiple tags, the ttl of each tag is processed separately.
 
 ## Setting a TTL Value
 
-Setting a TTL value allows you to specify the living time of the data.
+- Setting a TTL value for the existed tag.
+
+```ngql
+nebula> CREATE TAG t1(a timestamp);
+nebula> ALTER TAG t1 ttl_col = "a", ttl_duration = 5; -- Setting ttl
+nebula> INSERT VERTEX t1(a) values 101:(now());
+```
 
-Creating a tag then adding TTL attribute.
+The vertex 101 property in tag t1 will expire after 5 seconds since specified by now().
+
+- Or you can set the TTL attribute when creating the tag.
 
 ```ngql
-nebula> CREATE TAG t(a int, b int, c string);
-nebula> ALTER TAG t ADD ttl_col = "a", ttl_duration = 1000; -- add ttl attribute
-nebula> SHOW CREATE TAG t;
+nebula> CREATE TAG t2(a int, b int, c string) ttl_duration= 100, ttl_col = "a";
+nebula> INSERT VERTEX t2(a, b, c) values 102:(1584441231, 30, "Word");
 ```
 
-Or you can set the TTL attribute when creating the tag.
+The vertex 102 property in tag t2 will expire after 100 seconds since 18:33:51 CSTMarch 17, 2020 (MacOS).
+
+- When a vertex has multiple TAGs, the TTL of each TAG is independent from each other.
 
 ```ngql
-nebula> CREATE TAG t(a int, b int, c string) ttl_duration= 100, ttl_col = "a";
+nebula> CREATE TAG t3(a string);
+nebula> INSERT VERTEX t1(a),t3(a) values 200:(now(), "hello");
 ```
 
-## Dropping TTL
+The vertex 200 property in tag t1 will expire after 5 seconds.
 
-If you have set a TTL value for a field and later decide do not want it to ever automatically expire, you can drop the TTL value or invalidate it. For example, using the previous example.
+```ngql
+nebula> FETCH PROP ON t1 200;
+Execution succeeded (Time spent: 5.945/7.492 ms)
+
+nebula> FETCH PROP ON t3 200;
+======================
+| VertexID | t3.a    |
+======================
+| 200      | hello   |
+----------------------
+
+nebula> FETCH PROP ON * 200;
+======================
+| VertexID | t3.a    |
+======================
+| 200      | hello   |
+----------------------
+```
+
+## Dropping TTL
 
-Drop the TTL attribute:
+If you have set a TTL value for a field and later decide do not want it to ever automatically expire, you can drop the TTL value, set it to null or invalidate it by setting it to 0 or -1.
```

Comment:
If `ttl_col` is configured to an empty string, the item will not expire.
If the tag does not contain `ttl_col` configured field, the item will not expire.
If `ttl_duration` is set to 0 or -1, the item will not expire.

Reply:
If you have set a TTL value for a field and later decide do not want it to ever automatically expire, you can drop the TTL value, set it to an empty string or invalidate it by setting it to 0 or -1.